Wrapping/Python/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ include_directories ( ${SimpleITK_INCLUDE_DIRS} ${CMAKE_CURRENT_SOURCE_DIR} )
 #
 # Options
 #
+option ( SimpleITK_PYTHON_FLATSTATICMETHOD "Enable object_method functions for static class members. \
+This feature will be removed in the future. To disable requires SWIG>=4.1.0" ON )
```

Comment:
Possibly add an example of a flat static method to the help text so that others understand what this means. Change to :"... for static class members (e.g. ImageSeriesReader_GetGDCMSeriesIDs)."

Reply:
The "object_method" was intended to be an example.